client/extensions/woocommerce/app/order/order-details/fee-dialog.js

```diff
@@ -107,7 +107,7 @@ class OrderFeeDialog extends Component {
 		const dialogButtons = [
 			<Button onClick={ closeDialog }>{ translate( 'Cancel' ) }</Button>,
 			<Button primary onClick={ this.handleFeeSave } disabled={ ! canSave }>
-				{ translate( 'Add Fee' ) }
+				{ translate( 'Add fee' ) }
```

Comment:
:disappointed: 1 existing translations will be lost with this change.
If the string must change, the following option has already been translated **1** time:
`translate( 'Add a fee' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).